### ISSUE TITLE
Live audio input

### ISSUE THREAD

**Issue by balandinodidonato** [LibPdIntegration/LibPdIntegration#28]
Is there a way to have a live audio input (i.e. computer microphone) fed into the Pd patch running in Unity?

**Comment by thomasvanta** [comment]
Yes, if you use units microphone class to attach a microphone to the AudioSource and use adc in pd, you'll effectively get audio from the microphone fed into Pd.
